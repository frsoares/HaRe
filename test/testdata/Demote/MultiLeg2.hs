module Demote.MultiLeg where

data Constraint = Include String
                | Exclude String
               deriving (Show)

is_excluded stringQueueName (Include regex) = do
  reg <- mkr regex
  return True
is_excluded stringQueueName (Exclude regex) = do
  -- reg <- mkr regex
  return False

mkr reg = show reg
