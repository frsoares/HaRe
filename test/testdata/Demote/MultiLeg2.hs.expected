module Demote.MultiLeg where

data Constraint = Include String
                | Exclude String
               deriving (Show)

is_excluded stringQueueName (Include regex) = do
  reg <- mkr
  return True
    where
       mkr     = show regex
    

is_excluded stringQueueName (Exclude regex) = do
  -- reg <- mkr regex
  return False

